scripts/firefox-watch.js

```diff
@@ -0,0 +1,18 @@
+const nodemon = require("nodemon");
+const fs = require("fs");
+
+const FILES_TO_WATCH = "ts,tsx,scss";
+
+const watcher = nodemon({
+  ext: FILES_TO_WATCH,
+  script: "firefox-build-manifest.js",
```

Comment:
File was moved, should be updated to:

```suggestion
  script: "scripts/firefox-build-manifest.js",
```